src/ring_theory/power_series/well_known.lean

```diff
@@ -83,4 +84,74 @@ variables {A A'} (n : ℕ) (f : A →+* A')
 
 end field
 
+open ring_hom
+open finset nat
+
+variables (A : Type*) [comm_ring A]
```

Comment:
```suggestion
variables {A : Type*} [comm_ring A]
```
which will require changes below

Reply:
Is there any reason for not wanting it to be explicit?

Reply:
Yes, because when you use `eval_mul_hom A a`, the `A` is always knowable from the `a` that follows it.

Reply:
Ah I see, I will just change it, thank you!